includes/datavalues/SMW_DV_Number.php

```diff
@@ -66,7 +73,7 @@ class SMWNumberValue extends SMWDataValue {
 	 * @return integer 0 (no errors), 1 (no number found at all), 2 (number
 	 * too large for this platform)
 	 */
-	static protected function parseNumberValue( $value, &$number, &$unit ) {
+	static protected function parseNumberValue( $value, &$number, &$unit, &$asPrefix = false ) {
```

Comment:
Well, the whole thing needs a general overhaul but I'm not in a mood to bite the bullet yet (for once I want to add some new things not re-factor legacy stuff).